src/Trace/TraceClient.php

```diff
@@ -0,0 +1,192 @@
+class TraceClient
+{
+    use ArrayTrait;
+    use ClientTrait;
+
+    const FULL_CONTROL_SCOPE = 'https://www.googleapis.com/auth/cloud-platform';
+    const READ_ONLY_SCOPE = 'https://www.googleapis.com/auth/trace.readonly';
+
+    /**
+     * @var ConnectionInterface $connection Represents a connection to Trace
+     */
+    protected $connection;
+
+    /**
+     * Create a Trace client.
+     *
+     * @param array $config [optional] {
+     *     Configuration options.
+     *
+     *     @type string $projectId The project ID from the Google Developer's
+     *           Console.
+     *     @type CacheItemPoolInterface $authCache A cache used storing access
+     *           tokens. **Defaults to** a simple in memory implementation.
+     *     @type array $authCacheOptions Cache configuration options.
+     *     @type callable $authHttpHandler A handler used to deliver Psr7
+     *           requests specifically for authentication.
+     *     @type callable $httpHandler A handler used to deliver Psr7 requests.
+     *           Only valid for requests sent over REST.
+     *     @type array $keyFile The contents of the service account credentials
+     *           .json file retrieved from the Google Developer's Console.
+     *           Ex: `json_decode(file_get_contents($path), true)`.
+     *     @type string $keyFilePath The full path to your service account
+     *           credentials .json file retrieved from the Google Developers
+     *           Console.
+     *     @type int $retries Number of retries for a failed request.
+     *           **Defaults to** `3`.
+     *     @type array $scopes Scopes to be used for the request.
+     * }
+     */
+    public function __construct(array $config = [])
+    {
+        if (!isset($config['scopes'])) {
+            $config['scopes'] = [self::FULL_CONTROL_SCOPE];
+        }
+
+        $this->connection = new Rest($this->configureAuthentication($config));
+    }
+
+    /**
+     * Sends a Trace log in a simple fashion.
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects/patchTraces Project patchTraces API documentation.
+     * @codingStandardsIgnoreEnd
+     *
+     * @param Trace $trace The trace log to send.
+     * @param array $options [optional] Configuration Options
+     * @return bool
+     * @throws ServiceException
+     */
+    public function insert(Trace $trace, array $options = [])
+    {
+        return $this->insertBatch([$trace], $options);
+    }
+
+    /**
+     * Sends multiple Trace logs in a simple fashion.
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects/patchTraces Project patchTraces API documentation.
+     * @codingStandardsIgnoreEnd
+     *
+     * @param Trace[] $traces The trace logs to send.
+     * @param array $options [optional] Configuration Options
+     * @return bool
+     * @throws ServiceException
```

Comment:
I noticed this tag was omitted for some of the other methods that trigger network requests. Should we leave these ones out too for consistency?